public/client.js

```diff
@@ -54,25 +54,15 @@ function backgroundPosition(idx) {
 // Position the image in the right place on each tile to reassemble it on the grid, and enable click events on the tiles
 function tileSetup() {
   const tileArray = document.querySelectorAll(".tile");
-  const gameArea = document.getElementById("game-area");
```

Comment:
Assume that deleting this line was a mistake, right?

Reply:
Nope, it was defined but wasn't used in the code.